caravel/assets/javascripts/explorev2/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ExploreViewContainer from './components/ExploreViewContainer';
+
+import { createStore, applyMiddleware, compose } from 'redux';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import { getDevEnhancer } from '../../utils/common';
```

Comment:
i moved this and other redux utils to  a common reductUtils.js file `https://github.com/ascott/caravel/blob/master/caravel/assets/javascripts/reduxUtils.js#L67`